src/test/testRunner.ts

```diff
@@ -127,10 +127,12 @@ class CoverageRunner {
             // On Windows, extension is loaded pre-test hooks and this mean we lose
             // our chance to hook the Require call. In order to instrument the code
             // we have to decache the JS file so on next load it gets instrumented.
-            // This doesn"t impact tests, but is a concern if we had some integration
-            // tests that relied on VSCode accessing our module since there could be
-            // some shared global state that we lose.
-            decache(fullPath)
+            // This breaks tests that depend on global state, e.g. any test
+            // that depends on implicit VSCode features (launch.json processing,
+            // LSP symbols, ...).
+            if (os.platform() === 'win32') {
```

Comment:
Out of curiosity, did this create any adverse behaviors on non-Windows platforms?

Reply:
It definitely creates adverse behavior (described in the comment) which is why this guard was added. 

But removing decache() call seems to prevent coverage reports so this change will be reverted. https://github.com/aws/aws-toolkit-vscode/pull/1315#discussion_r496134053